Platform/Shared/VMDetailsView.swift

```diff
@@ -93,6 +93,14 @@ struct Screenshot: View {
     let large: Bool
     @EnvironmentObject private var data: UTMData
     
+    @State private var commandDown = false
+    
+    #if os(macOS)
+    private var playImageName: String { commandDown ? "play.circle" : "play.circle.fill" }
```

Comment:
same as above; use `play.rectangle` for play-as-snapshot; meaning for filled use `play.rectangle.fill`

Also please change it to option instead of command key. In many apps incl. Finder you get alternative features from buttons while holding option/alt. ⌥